packages/react-components/react-nav/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@fluentui/react-nav",
+  "version": "9.0.0-alpha.0",
```

Comment:
This should be `@fluentui/react-nav-preview` and start at version `0.0.0`, according to https://github.com/microsoft/fluentui/wiki/new-release-process---v9-packages.

The create-package script should have added `-preview` automatically, based on the changes in https://github.com/microsoft/fluentui/pull/28474. Did that not happen?

cc @Hotell